src/index.tsx

```diff
@@ -233,13 +278,14 @@ class PyflyByWidget extends Widget {
             const currentLockId = this._lock.requestedLockCount;
             this._lock.enable(currentLockId + 1);
```

Comment:
I know this code was not changed, but reading it is throwing me a bit since it's not clear what it is enabling. I think we should rename `enable` to `acquire` and return a promise. So these lines become:
```
const currentLockId = await this._lock.aquire(); # Waits for the previous lock to expire and returns a promise that resolves to it's lock id
```

Then later:
```
this._lock.release(lockId)
```

Reply:
Agreed